weave-js/src/components/PagePanelComponents/Home/HomePreviewSidebar.tsx

```diff
@@ -66,8 +68,7 @@ export const HomePreviewSidebarTemplate: React.FC<{
               cursor: 'pointer',
             }}
             onClick={e => {
-              e.stopPropagation();
-              props.setPreviewNode(undefined);
+              history.push('.');
```

Comment:
what is the effect of `.`?

Reply:
It would take you to the "current directory", so 
`/wandb/<entity>/<project>/tables/some_table`
would navigate to
`/wandb/<entity>/<project>/tables/`

Reply:
👍 nice - very clean